server/index.js

```diff
@@ -14,36 +15,20 @@ app.use(express.static(__dirname + '/../react-client/dist'));
 
 
 app.get('/images/urls/:itemId', function (req, res) {
-  // console.log(req.params.itemId);
-  Images.fetchItemImages(req.params.itemId)
+  Images.fetchItemImages(Number(req.params.itemId))
     .then((data) => {
       if (data) {
```

Comment:
Good structure for error handling! 

Also, it's not going to work. You're checking if data is truthy. But if it doesn't find a value the `find` function returns an empty array which is... truthy! so you'll be sending an object with a data property which has an empty array. Not the end of the world, but not what you intended.

![Screen Shot 2020-10-02 at 9 56 18 PM](https://user-images.githubusercontent.com/279936/94983789-e5e11280-04fa-11eb-8088-f454b0857d00.png)

Reply:
Ahh that makes sense! I'll change it.